pkg/security/podsecuritypolicy/capabilities/capabilities.go

```diff
@@ -36,11 +37,23 @@ var _ Strategy = &defaultCapabilities{}
 
 // NewDefaultCapabilities creates a new defaultCapabilities strategy that will provide defaults and validation
 // based on the configured initial caps and allowed caps.
-func NewDefaultCapabilities(defaultAddCapabilities, requiredDropCapabilities, allowedCaps []api.Capability) (Strategy, error) {
+func NewDefaultCapabilities(defaultAddCapabilities, requiredDropCapabilities, allowedCaps []corev1.Capability) (Strategy, error) {
+	internalDefaultAddCaps := make([]api.Capability, len(defaultAddCapabilities))
+	for i := 0; i < len(defaultAddCapabilities); i++ {
```

Comment:
this form would be easier to read:

```suggestion
	for i, c := range defaultAddCapabilities {
		internalDefaultAddCaps[i] = api.Capability(c)
	}
```